lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -6,8 +6,9 @@ def define
       status :published
       satisfies_need "100696"
 
-      benefit_cap_query = SmartAnswer::Calculators::BenefitCapCalculatorDataQuery.new
-      benefit_cap_rates = benefit_cap_query.data.fetch('rates')
+      query = Calculators::BenefitCapCalculatorDataQuery.new
+      rates = query.data.fetch('rates')
+      benefits = query.data.fetch('benefits')
```

Comment:
I think the benefits added to benefit_cap_data.yml in "Update data structure for benefit cap rates and add list of benefits" should've been added in this commit (i.e. at the point they're extracted from the flow).